ansible/group_vars/lab/lab.yml

```diff
@@ -1,15 +1,34 @@
 ---
-#starlab (str) group variables
-# file: group_vars/str.yml
+#testlab (lab) group variables
+# file: group_vars/lab.yml
 
 # ntp variables
 ntp_servers: ['10.0.0.1', '10.0.0.2']
 
 # syslog variables
 syslog_servers: ['10.0.0.5', '10.0.0.6']
-#
+
 # dns variables
 dns_servers: ['10.0.0.5', '10.0.0.6']
+
+# forced_mgmt_routes
+forced_mgmt_routes: ['10.0.0.100/31', '10.250.0.8', '10.255.0.0/28']
```

Comment:
I'm not sure here: It should be '10.250.0.8' (without prefix), or '10.250.0.8/32'